draft-ietf-quic-ack-frequency.md

```diff
@@ -150,8 +150,10 @@ sending the following transport parameter ({{Section 7.2 of QUIC-TRANSPORT}}):
 min_ack_delay (0xff03de1a):
 
 : A variable-length integer representing the minimum amount of time in
-  microseconds by which the endpoint can delay an acknowledgement. This limit
-  could be based on the receiver's clock or timer granularity.
+  microseconds by which the endpoint that is sending this vlaue is able to
+  delay an acknowledgement. This limit could be based on the receiver's clock
```

Comment:
/limit/ ? I suggest /value/
... OR if you prefer /this value could be limited by .../